fvtest/compilertriltest/ArithmeticTest.cpp

```diff
@@ -808,6 +816,227 @@ INSTANTIATE_TEST_CASE_P(ArithmeticTest, FloatArithmetic, ::testing::Combine(
         std::make_tuple<const char*, float (*)(float, float)>("fdiv", static_cast<float (*)(float, float)>(div))
     )));
 
+#ifdef OMR_ENV_DATA64
+class UInt64Int64Arithmetic : public TRTest::OpCodeTest<uint64_t, uint64_t, int64_t> {};
```

Comment:
The name for these tests should be `AddressInt64Arithmetic`, because that is what's being tested. The fact that it's using uint64_t for the address type is just a detail of the the test implemented. Can you also add a comment explaining why uint64_t is used for the address type?